monero-outreach-docs/translations/es/PSA/files/mining/mine_support_network_es.md

```diff
@@ -0,0 +1,27 @@
+
+Sin embargo, se ha asumido que el valor intrínseco o la rentabilidad general de la minería de Monero empieza a ser notable, más individuos y empresas intentarán configurar los ASICs y los FPGAs para tener una ventaja comparativa y llevarse las recompensas de la minería en contra de los mineros CPU/GPU con un hashrate menor. Algunos miembros de la comunidad les preocupa el riesgo sobre un ataque del 51% en Monero de parte de los ASICs y los FPGAs con su alto poder de procesamiento hash que sus máquinas pueden ofrecer.
+
+Como resultado de esto, hay un *llamado a la acción* para defender y asegurar la red de Monero *para minar con cualquier medio posible que se tenga a disposición*. Te animamos para que te conviertas en un participante activo en la red de Monero y ayudes a asegurarla. 
+
+¡Únete a la lucha para proteger el efectivo privado! He aquí unas opciones prácticas que podrías considerar para ayudar a proteger la red:
+
+1. **Minería en solitario (Solo Mining)-** Esto es visto como la opción de la lotería en la minería. Si prendes el minador en el CPU o GPU de tu computadora para minar Monero, estarás apostando que ganarás algún bloque frente a todos los otros mineros en la red. Recibirás el total de la recompensa de ese bloque, pero las probabilidades de que esto suceda realmente son bajas. Puedes minar en solitario desde los monederos oficiales GUI y CLI de Monero ([https://www.getmonero.org/es/resources/user-guides/solo_mine_GUI.html](https://www.getmonero.org/es/resources/user-guides/solo_mine_GUI.html)).
+
+2. **Minería en una Pool-** Este es uno de los medios comunes para minar Monero. Minar con una pool tiende a tener pagos regulares, pero son más pequeños, porque la recompensa se divide entre todos los que ayudaron a minar ese bloque. Aprende más en ([minexmr.com/#getting_started](https://minexmr.com/#getting_started)) y ([minexmr.com/pools.html](https://minexmr.com/pools.html)).
+
+3. **Minería Web-** Si eres dueño o tienes tu propio sitio web, puedes implementar la minería web y utilizar el poder de cómputo del visitante. Sitios web han comenzado a hacer esto de una manera ética en reemplazo a la publicidad e incluye una opción que el usuario puede optar por participar en la minería o no. No hagas minería web sin darle al usuario la opción de optar por minar. Aprende más en ([coinhive.com](https://coinhive.com/)). Aquí hay algunos ejemplos para minar con tu navegador web: [bailbloc.thenewinquiry.com/webminer.html](https://bailbloc.thenewinquiry.com/webminer.html) y [thehopepage.org](https://www.thehopepage.org/). De manera alternativa, puedes minar con tu propio navedador. Una manera de hacer esto es ir al tablero (dashboard) de Monero Ocean, desplazarse hacia abajo y hacer clic en el botón >Web Mining: [moneroocean.stream/#/dashboard](https://moneroocean.stream/#/dashboard).
```

Comment:
- _...e incluye una opción de participación para sus visitantes._ This is more a suggestion than a correction.

Reply:
I'll take the suggestion. Done.